specviz/widgets/workspace.py

```diff
@@ -132,6 +132,17 @@ def _on_toggle_visibility(self, state):
 
         self.proxy_model.dataChanged.emit(idx, idx)
 
+    def _on_changed_color(self, color):
```

Comment:
What is this method used for? There already is the method in `plotting.py` that handles changing the line color.